base_station/gui/src/components/ScoopUV.vue

```diff
@@ -102,6 +117,15 @@ export default {
         'device': id,
         'enable': enabled
       })
+    },
+    UVshutdown: function(status) {
+      this.setPart(this.mosfetIDs.uvBulb, status);
+      if (this.shutdown === 1) {
```

Comment:
So looking at this more, my concern is we could turn it on, turn it off, turn it on, and then the first timeout would turn it off in shorter than 2 minutes. Can you use the cleartimeout function to mitigate this?
https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout